presto-docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -787,7 +791,7 @@ dropping the table from the metadata catalog using ``TRUNCATE TABLE``.
 DELETE
 ^^^^^^^^
 
-The Iceberg connector can delete data in one or more entire partitions from tables by using ``DELETE FROM``. For example, to delete from the table ``lineitem``::
```

Comment:
Should we preserve the older version of the documentation for V1 tables?

Reply:
Reviewed, nice work!